build/Dockerfile.ci

```diff
@@ -0,0 +1,101 @@
+# CI Dockerfile
+# CI requires a special Dockerfile for use with OpenShift Builds which is what is used to build the image. This requires
+# building the operator from the PR source without using the operator-sdk.
+# NOTE: RUN mkdir <dir> is being used in cases where a directory needs to be created as CI image builds does not seem to
+# be creating the directory as a result of the WORKDIR directive. 
+# build stage for building binaries
+FROM registry.access.redhat.com/ubi8/go-toolset:1.12.8-32 as build
+LABEL stage=build
+
+# Build WMCB
+# WORKDIR is not resulting in creating this directory, so we are forcing it to happen. The image does not allow us to
+# create /build/, forcing us to use $HOME which is /opt/app-root/src/
+RUN mkdir $HOME/build/
+WORKDIR $HOME/build/
+RUN git clone https://github.com/openshift/windows-machine-config-bootstrapper.git
+WORKDIR windows-machine-config-bootstrapper
+RUN make build
+
+# Build hybrid-overlay
+WORKDIR $HOME/build/
+RUN git clone https://github.com/openshift/ovn-kubernetes/
+WORKDIR ovn-kubernetes/go-controller/
+RUN make windows
+
+# Build WMCO
+# The source here corresponds to the code in the PR and is placed here by the CI infrastructure.
```

Comment:
This is correct. Please look at [this](https://github.com/openshift/cluster-kube-descheduler-operator/commit/37bad185e8e8b2a11116cd8bb8cfa7d5b709d611#diff-3254677a7917c6c01f55212f86c57fbf). If we use src in build, we should get the entire source which is part of the PR.

Reply:
The issue I am trying to fix is with building the operator image and not building the binary. I do not know how to specify `src` there. Please see https://github.com/openshift/release/pull/7317/commits/73762f154f560a9f0e0112d7e2f4137e17f19e22. Is there a way to specify `src` there?